cli/server_test.go

```diff
@@ -201,7 +201,16 @@ func TestServer(t *testing.T) {
 		go func() {
 			errCh <- inv.WithContext(ctx).Run()
 		}()
-		pty.ExpectMatch("Using an ephemeral deployment directory")
+		matchCh1 := make(chan string, 1)
+		go func() {
+			matchCh1 <- pty.ExpectMatchContext(ctx, "Using an ephemeral deployment directory")
+		}()
+		select {
+		case err := <-errCh:
+			require.NoError(t, err)
+		case <-matchCh1:
+			// OK!
+		}
```

Comment:
If you have the matched string here, why not move the assertions from L214-220 in here?

Reply:
I wanted to keep modifications to the existing test to a minimum.